### PR TITLE
subverting rspec deprecation

### PR DESCRIPTION
**Pull request by rift137**
Deprecation Warnings:
Filtering by an `:example_group` subhash is deprecated. Use the subhash to filter directly instead.  ...

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -29,7 +29,7 @@ gem 'newrelic_rpm'
 gem 'pinglish'
 gem 'possessive'
 gem 'pretender', '~> 0.0.3'
-gem 'pundit'
+gem 'pundit', github: 'elabs/pundit'
```
Comment (mikestephens) [inline Gemfile]: why is this necessary?

Comment (halloffame) [inline Gemfile]: Why are you using the github version?

Comment (mikestephens) [inline Gemfile]: ^ +1

Comment (rift137) [inline Gemfile]: Because its magical and was suggested

Comment (halloffame) [inline Gemfile]: by whom may I ask

Comment (rift137) [inline Gemfile]: https://github.com/elabs/pundit/issues/189
